app/javascript/styles/components.scss

```diff
@@ -90,6 +90,21 @@
   }
 }
 
+.column-collapsable__button,
+.column-icon-clear {
+  background-color: $classic-base-color !important;
+  border-top-right-radius: 5px;
+  -moz-border-top-right-radius: 5px;
```

Comment:
It seems Mastodon does not use vendor prefixes for border-radius.
So, we can remove -moz-, -webkit- from border radius.
And Mastodon use border-radius rather than border-top-right-radus and so on. See: https://github.com/tootsuite/mastodon/blob/master/app/javascript/styles/accounts.scss#L255